src/openrct2/config/ConfigEnum.h

```diff
@@ -0,0 +1,79 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
`*.hpp` is when there are implementations in the header. Although I admit in this case there is due to the template. But hpp does not mean C++ header - it means a C++ header containing C++ implementation code.